ckan/lib/helpers.py

```diff
@@ -1520,11 +1511,39 @@ def gravatar(email_hash, size=100, default=None):
         default = quote(default, safe='')
 
     return literal('''<img src="//gravatar.com/avatar/%s?s=%d&amp;d=%s"
-        class="gravatar" width="%s" height="%s" alt="Gravatar" />'''
+        class="user-image" width="%s" height="%s" alt="Gravatar" />'''
```

Comment:
This css class doesn't have any effect since, by specificity, is override by `.context-info .image img` (see my final comment)